03/app.js

```diff
@@ -1,14 +1,41 @@
-console.log('DOM');
+console.log("DOM");
 
 const buttonSettings = {
-    attr: {
-        className: 'btn',
-        title: 'super button'
-    },
-    css: {
-        border: '1px solid #336699',
-        padding: '5px 20px',
-        color: '#444'
-    },
-    text: 'Click me!',
-}
\ No newline at end of file
+  attr: {
+    className: "btn",
+    title: "super button",
+  },
+  css: {
+    border: "1px solid #336699",
+    padding: "5px 20px",
+    color: "#444",
+  },
+  text: "Click me!",
+};
+
+const btnParent = document.querySelector(".parent-for-button");
+
+const btn = document.createElement("button");
+for (const setting in buttonSettings) {
+  switch (setting) {
+    case "attr":
+      for (const [key, value] of Object.entries(buttonSettings[setting])) {
+        if (key === "className") {
+          btn.classList.add(value);
```

Comment:
Zrobiło się to bardzo rozbudowane, czy nie lepiej zrobić 2x pętla dla każdej właściwości tj. `for(... in buttonSettings.attr)` itp,